deployment/ccip/changeset/token_info.go

```diff
@@ -20,9 +25,26 @@ const (
 	LinkDecimals             = 18
 	WethDecimals             = 18
 	UsdcDecimals             = 6
+	// MockLinkAggregatorDescription This is the description of the MockV3Aggregator.sol contract
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/tests/MockV3Aggregator.sol#L76-L76
+	MockLinkAggregatorDescription = "v0.8/tests/MockV3Aggregator.sol"
+	// MockWETHAggregatorDescription WETH use description from MockETHUSDAggregator.sol
```

Comment:
This comment is unintelligible, maybe 

```suggestion
	// MockWETHAggregatorDescription is the description from MockETHUSDAggregator.sol
```